ktx-library-codelab/build.gradle

```diff
@@ -45,23 +43,23 @@ buildscript {
     }
     repositories {
         google()
-        jcenter()
+        mavenCentral()
         maven {
-            url "http://dl.bintray.com/kotlin/kotlin-eap"
+            url "https://dl.bintray.com/kotlin/kotlin-eap"
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.2'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_versionsion"
+        classpath 'com.android.tools.build:gradle:7.0.4'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
         maven {
-            url "http://dl.bintray.com/kotlin/kotlin-eap"
+            url "https://dl.bintray.com/kotlin/kotlin-eap"
```

Comment:
might not be needed anymore?

Reply:
The same as for above: on my side it works with secured protocol only.

Reply:
Removed.